orchestrators/IntelligenceOperations/src/DTOs/AiStatusSnapshot.php

```
@@ -0,0 +1,149 @@
+
+    /**
+     * @param array<string, AiCapabilityStatus> $capabilityStatuses
+     * @return array<string, AiCapabilityStatus>
+     */
+    private function normalizeCapabilityStatuses(array $capabilityStatuses): array
+    {
+        $normalized = [];
+
+        foreach ($capabilityStatuses as $featureKey => $status) {
+            if (!is_string($featureKey) || trim($featureKey) === '') {
+                throw AiStatusContractException::invalidValue('AI capability status keys');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\IntelligenceOperations\Exceptions\AiStatusContractException' in method 'normalizeCapabilityStatuses'. (undefined)

(StaticAccess)